dvc/commands/plots.py

```diff
@@ -42,7 +42,7 @@ def _adjust_vega_renderers(renderers):
                         dp["rev"] = "::".join(vi.values())
             else:
                 for dp in r.datapoints:
-                    dp.pop(VERSION_FIELD, {})
+                    dp[REVISION_FIELD] = dp[VERSION_FIELD]["revision"]
```

Comment:
Is there a reason to change the format of the "non json" output to now include the full `VERSION_FIELD` dict?

```suggestion
                    dp[REVISION_FIELD] = dp[VERSION_FIELD]["revision"]
                    dp.pop(VERSION_FIELD, {})
```